src/chainparams.cpp

```diff
@@ -202,6 +221,22 @@ class CTestNetParams : public CMainParams {
             0
         };
 
+        // Founders reward script expects a vector of 2-of-3 multisig addresses
+        vFoundersRewardAddress = {
+            "2N2e2FRfP9D1dRN1oRWkH7pbFM69eGNAuQ4", "2NBW8WsA2jUussoJbRv82UXH1BYopkjYqcd", "2N1MudZmwDFTcYiLCZfrcsnhHwaSTTigbcN", "2MxfUJXWKz9D8X3mcMpVcdEJKdJ6zFukca9",
```

Comment:
https://github.com/zcash/zcash/pull/1398/files#diff-eb0f2e0254c41a80ca078306281d2515R20

The mainnet addresses may be generated in a different way that facilitates keeping them off-line, but we don't discuss detailed operational security issues on github.